src/main/java/hlf/java/rest/client/util/FabricChannelUtil.java

```diff
@@ -0,0 +1,174 @@
+package hlf.java.rest.client.util;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.hyperledger.fabric.protos.common.Configtx;
+import org.hyperledger.fabric.protos.common.MspPrincipal;
+import org.hyperledger.fabric.protos.common.Policies;
+
+@Slf4j
```

Comment:
[nitpick] you can safely remove this if nothing is being logged in this class, or can introduce log statements if required.

Reply:
Addressed, removed it.